src/deadline/client/cli/_groups/job_group.py

```diff
@@ -396,6 +396,19 @@ def _download_job_output(
     # in Job Attachments library can be increased (currently using default number, 10, which
     # makes it keep logging urllib3 warning messages when downloading large files)
     with _modified_logging_level(logging.getLogger("urllib3"), logging.ERROR):
+
+        @api.record_success_fail_telemetry_event(metric_name="download_job_output")  # type: ignore
```

Comment:
Looks like here, and other uses of this decorator we have to disable type checking. Is there alternative to this?

Reply:
No - it is due to mac mypy issues. It passes on all other operating systems.